src-plugins/itkFilters/itkMorphologicalFiltersProcessBase.cpp

```diff
@@ -34,29 +44,15 @@ itkMorphologicalFiltersProcessBase::itkMorphologicalFiltersProcessBase(itkMorpho
     d->radiusMm[1] = 0;
     d->radiusMm[2] = 0;
 
-    d->isRadiusInPixels = false;    
-    d->description = "";
-
-    d->kernelShape = BallKernel;
-}
+    d->isRadiusInPixels = false;
 
-itkMorphologicalFiltersProcessBase::itkMorphologicalFiltersProcessBase(const itkMorphologicalFiltersProcessBase& other) 
-    : itkFiltersProcessBase(*new itkMorphologicalFiltersProcessBasePrivate(*other.d_func()), other)
-{
-    
-}
-
-itkMorphologicalFiltersProcessBase::~itkMorphologicalFiltersProcessBase()
-{
-    
+    d->kernelShape = itkMorphologicalFiltersProcessBase::BallKernel;
```

Comment:
we find here the initialization of variables. We enter now these constructors. `kernelShape` and "isRadiusInPixels" for instance are initialized (there were our original errors)